src/styles/_section_course_curriculum.scss

```diff
@@ -1,97 +1,284 @@
 // Course Curriculum
 // ======================================
 
-.course-curriculum {
-  @include section-height-preset($section-height__medium);
+[class*="course-curriculum"] {
 
   .section__heading {
     margin: 0 0 $gutter-width * 1.5;
   }
 }
 
-.course-curriculum__chapter-list {
-  list-style: none;
-}
+.course-curriculum {
+  @include section-height-preset($section-height__large);
 
-.course-curriculum__chapter {
-  margin: 0 0 $gutter-width * 1.5;
-}
+  .course-curriculum__chapter-list {
+    @include make-row;
 
-.course-curriculum__chapter-header {
-  @include make-row;
+    .course-curriculum__chapter {
+      @include screen-md-min {
+        padding-right: $gutter-width * 2;
+      }
 
-  position: relative;
-  margin-bottom: $gutter-width / 2;
-  border-bottom: 1px solid $gray-1;
-  cursor: pointer;
+      padding-right: $gutter-width;
+      padding-left: 0;
+    }
+  }
 
-  &.collapse {
+  .course-curriculum__chapter {
+    @include screen-lg-min {
+      @include make-col(6);
+    }
+    @include make-col(12);
+    @include display(flex);
 
-    i {
-      &:after {
-        @include transform(translate(2px, 2px) rotate(-140deg));
+    padding-top: $gutter-width;
+    padding-bottom: $gutter-width;
+    border: $interface__border;
+    line-height: 1;
+  }
+
+  .course-curriculum__chapter-number {
+    padding: 0 $gutter-width;
+    color: transparentize($heading__font-color, .8);
```

Comment:
I'd prefer if we had leading 0's, haha, not sure what your thoughts are

Reply:
I always thought the leading 0s were bad. Mostly for saving a few bytes here and there... but the linter defaults were like that (I think?). Also, in reference to that REM calculation bug from earlier... I later noticed that the linter wasn't actually picking it up. It was throwing an error for extra space elsewhere. I was just looking at the Warning symbol and assuming that's what it was because it would do it elsewhere. Maybe it's not such a huge deal. I'm not tied to it either way.

Reply:
Since the REM calculation requires the leading 0, and I'll be going forward with REM, for consistency, I'll be updating everything to include leading 0.